wire-ios/Wire-iOS/Sources/UserInterface/Settings/ColorPickerView.swift

```diff
@@ -0,0 +1,83 @@
+//
+// Wire
+// Copyright (C) 2024 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import SwiftUI
+import WireCommonComponents
+import WireSyncEngine
+
+struct ColorPickerView: View {
+
+    @State var selectedColor: AccentColor?
+
+    let colors: [AccentColor]
+    var onColorSelect: ((AccentColor) -> Void)?
+
+    private let colorViewSize: CGFloat = 28
+    private let colorViewCornerRadius: CGFloat = 14
+
+    var body: some View {
+        NavigationView {
+            List(colors, id: \.self) { color in
+                HStack {
+                    // Color view
+                    Circle()
+                        .fill(Color(uiColor: color.uiColor))
+                        .frame(width: colorViewSize, height: colorViewSize)
+                        .padding(.trailing, 10)
+                    Text(color.name)
+
+                    Spacer()
+
+                    // Checkmark view
+                    if selectedColor == color {
+                        Image(systemName: "checkmark")
+                            .foregroundColor(Color(SemanticColors.Icon.foregroundDefaultBlack))
+                    }
+                }
```

Comment:
suggestion: move this to a private view builder function, something like:

```swift
@ViewBuilder
private func cell(for color: AccentColor) -> some View {
  // ...
}
```

In my opinion this makes it more readable when glancing at the `body`:

```swift
List(colors, id: \.self) { color in
  cell(for: color)
    .onTapGesture { ... }
}
```

Reply:
Done: https://github.com/wireapp/wire-ios/commit/e96b58d70b88e6a8fb4fbafbbe3da6cd5dc0f37b